libs/community/tests/integration_tests/retrievers/document_compressors/test_embeddings_filter.py

```diff
@@ -41,3 +57,24 @@ def test_embeddings_filter_with_state() -> None:
     actual = relevant_filter.compress_documents(docs, query)
     assert len(actual) == 1
     assert texts[-1] == actual[0].page_content
+
+
+@pytest.mark.asyncio
```

Comment:
```suggestion
```

Reply:
if I delete this annotation, the test is not run.

Reply:
if I delete this annotation, the test is not run.